### DISCUSSION TITLE
（题外话）话说管理员们考不考虑建个Q群？

### DISCUSSION THREAD

**Discussion by TimG233** [Nemo2011/bilibili-api#674] [General]
有一些问题啊，或者交流什么的感觉用个Q群对一些人会方便些。github的issue有一种overkill的感觉。
比如隔壁网易云的repo(binaryify大佬的）就有个Q群挺方便的。
所以好奇这边的各位大佬们考不考虑整一个哈哈哈

**Comment by z0z0r4** [comment]
@Nemo2011

**Reply by z0z0r4** [reply]
> github的issue有一种overkill的感觉

这个还好吧...issue起码还能记录下，免得我两周回一次家，在学校看的啥都忘了

要是 QQ 群就真全忘了，issue 还能翻下

~~为什么不开 project...~~

**Reply by TimG233** [reply]
哈哈哈哈哈哈，可能回家过年了

**Reply by z0z0r4** [reply]
谁天天看GitHub...等着呗，也没几个人开发，就三，少一个群都凑不起来

biliup 倒是有 706750236
BAC 也有 560304737，但没人讨论蜀黍的破事就是了，who cares

~~网易云我咋记得 DMCA 了来着？~~

**Reply by TimG233** [reply]
坏了真DMCA了？这两天的事应该，不是吧，网易你坏事做尽
不是DMCA，只是警告邮件

**Reply by TimG233** [reply]
哦哦我在群里看到网易给Binaryify大佬的邮件了，6

**Reply by adk23333** [reply]
> 谁天天看GitHub...等着呗，也没几个人开发，就三，少一个群都凑不起来

可以考虑开个QQ频道，自由进出观看那种，反正也没几个人，频道的交流频率限制约等于没有。

我有贴吧和频道的管理经验

![Screenshot_2024-04-14-03-51-47-325_com.tencent.mobileqq.png](https://github.com/Nemo2011/bilibili-api/assets/84434845/3427ff2c-eb04-434a-b485-c4dbeeb253b1)

**Reply by z0z0r4** [reply]
no one cares

**Comment by z0z0r4** [comment]
https://t.me/bilibili_api 基本上我在校只看电报
